src/main/java/org/embulk/input/gcs/GcsFileInputPlugin.java

```diff
@@ -23,6 +25,7 @@ public class GcsFileInputPlugin
             .addDefaultModules().build();
     public static final ConfigMapper CONFIG_MAPPER = CONFIG_MAPPER_FACTORY.createConfigMapper();
     public static final TaskMapper TASK_MAPPER = CONFIG_MAPPER_FACTORY.createTaskMapper();
+    private Logger logger = LoggerFactory.getLogger(GcsFileInputPlugin.class);
```

Comment:
We usually have the logger instance as a class variable.

```suggestion
    private static final Logger logger = LoggerFactory.getLogger(GcsFileInputPlugin.class);
```